docs/Reference/Server.md

```diff
@@ -1568,6 +1568,22 @@ if (statusCode >= 500) {
   log.error(error)
 }
 ```
+
+When you are sending streams you will have to account for that in the custom
+error handler.
+
+If you use `res.send(jsonData)` in the error handler it will fail for stream
```

Comment:
We don't use "you" in the docs

Example:

```suggestion
If `res.send(jsonData)` is used in the error handler, it will fail for stream
```

Reply:
Oh, I referenced some docs in Reply that use `you`. I'll update.

Reply:
updated

Reply:
Yeah, we're trying to slowly unify everything under one style

There are some inconsistencies at the moment, and we're not fully there...